src/main/java/net/glowstone/inventory/WindowClickLogic.java

```diff
@@ -76,17 +77,18 @@ public static ClickType getClickType(int mode, int button, int slot) {
                 }
                 break;
 
-            case 5: // drag
-                break;
-
             case 6:
                 return ClickType.DOUBLE_CLICK;
+            case 5: // drag
```

Comment:
Is this correct or is it just not implemented yet?

Reply:
Marked as TODO to be safe.